basyx.components.databridge/Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM openjdk:11-slim-bullseye
 
+RUN apt update && apt install -y wget && apt clean
+
 # Copy built jar to image using the jar name specified in the pom.xml (JAR_FILE)
 ARG JAR_FILE
 
 COPY target/${JAR_FILE} /usr/share/databridgeExecutable.jar
 COPY target/lib /usr/share/lib
 
+EXPOSE 8085
```

Comment:
Please add a comment here that this is fore health check

Reply:
Added the comment in latest commit.